summit-demo/README.md

```diff
@@ -0,0 +1,181 @@
+SRPM is /home/tt/g/packit-service-testing-organisation/source-git-rpm/rpm-4.14.2-37.g824233d8.fc31.src.rpm
+SRPM: /home/tt/g/packit-service-testing-organisation/source-git-rpm/rpm-4.14.2-37.g824233d8.fc31.src.rpm
+```
+
+Werks, nice!
+
+```
+$ git remote add sg ssh://git@git.stg.centos.org/source-git/rpm.git
+```
+
+Let's not push to master just yet.
+
+```
+$ git checkout -B preview
+Switched to a new branch 'preview'
+
+$ git push -u sg
+The authenticity of host 'git.stg.centos.org (8.43.84.204)' can't be established.
+RSA key fingerprint is SHA256:e0tkihPvgWKC/ull50CaD4i40cYqPEOhVzW8jKaAa0M.
+Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
+Warning: Permanently added 'git.stg.centos.org,8.43.84.204' (RSA) to the list of known hosts.
+Enumerating objects: 1367, done.
+Counting objects: 100% (1367/1367), done.
+Delta compression using up to 8 threads
+Compressing objects: 100% (774/774), done.
+Writing objects: 100% (1367/1367), 5.03 MiB | 1.65 MiB/s, done.
+Total 1367 (delta 486), reused 1366 (delta 486)
+remote:   - to mqtt
+remote: ERROR: [u'git', u'rev-list', u'824233d87b25e8fa7211079f561c79baf34b3fa7', u'^HEAD'] =-- 128
+remote:
+remote: fatal: bad revision '^HEAD'
+remote:
+remote: Sending to redis to send commit notification emails
+To ssh://git.stg.centos.org/source-git/rpm.git
+ * [new branch]      preview -> preview
+Branch 'preview' set up to track remote branch 'preview' from 'sg'.
+```
+
+
+## Create a PR
+
+I did this via web ui: https://git.stg.centos.org/source-git/rpm/pull-request/1
+
+
+## Build
+
+Build kicked off locally using `copr-cli`:
+```
+$ packit srpm
+$ copr build ttomecek/centos-stream-rpm ./rpm-4.14.2-37.g8d29d36c.fc31.src.rpm
+```
+
+## Update check status
+
+We need to change the api token so that we can use packit's identity:
+```
+In [21]: s.change_token('the-token')
+```
+
+Sadly the commit check status (flag in pagure terminology) did not show up:
+```
+In [22]: cs = p.set_commit_status("8d29d36c13cf23a02d66789edb7ff735208b7ddd", CommitStatus.pending, "https://copr.fedorainfracloud.org/coprs/build/1315419", "RPM build", "The build is pending")
+
+In [24]: cs.comment
+Out[24]: 'RPM build'
+
+In [25]: cs.commit
+Out[25]: '8d29d36c13cf23a02d66789edb7ff735208b7ddd'
+
+In [26]: cs.context
+Out[26]: 'The build is pending'
+
+In [27]: cs.state
+Out[27]: <CommitStatus.pending: 1>
+
+In [28]: cs.url
+Out[28]: 'https://copr.fedorainfracloud.org/coprs/build/1315419'
+
+In [38]: css = p.get_commit_statuses("8d29d36c13cf23a02d66789edb7ff735208b7ddd")
+
+In [39]: css
+Out[39]:
+[<ogr.services.pagure.flag.PagureCommitFlag at 0x7f242f5f19d0>,
+ <ogr.services.pagure.flag.PagureCommitFlag at 0x7f242f506190>]
+```
+
+Instead, I posted a comment to the PR on packit's behalf:
+```
+In [36]: msg = f"RPM build [has finished](https://copr.fedorainfracloud.org/coprs/build/1315419).\nStatus: **succeeded**"
```

Comment:
For the presentation sake, maybe it would make sense to craft this message so that includes instructions for how to install the the rpms that were built.

Reply:
https://github.com/packit-service/research/pull/39/files#diff-a2bacf42a554f025dddcf4518e42313bR175

I'll try to do it in a VM today.